src/internal/FluencePeer.ts

```diff
@@ -71,7 +74,12 @@ export interface PeerConfig {
 /**
  * Information about Fluence Peer connection
  */
-interface ConnectionInfo {
+export interface PeerStatus {
+    /**
+     * Is the peer connected to network or not
+     */
+    isInitialized: Boolean;
```

Comment:
It's strange that PeerStatus contains both `isInitialized` and `isConnected`. It makes it possible to have `isInitialized: false, isConnected: true`.

Is there a way to describe it in a way that makes such a situation impossible?

Reply:
The current implementation is the easiest and most straightforward one. More complex implementation will require substantially more difficult checks for the end-user